lib/querystring.js

```diff
@@ -76,21 +84,21 @@ function unescapeBuffer(s, decodeSpaces) {
   var hasHex = false;
   while (index < s.length) {
     currentChar = s.charCodeAt(index);
-    if (currentChar === 43 /* '+' */ && decodeSpaces) {
-      out[outIndex++] = 32; // ' '
+    if (currentChar === CHAR_PLUS /* '+' */ && decodeSpaces) {
```

Comment:
I think a lot of the comments can be removed with this change since the constant names are much more self documenting than the character codes.

Reply:
I used to think so but not absolutely sure about that🤣.